assets/_js/custom-viz/boeing/waypoints.js

```diff
@@ -0,0 +1,7 @@
+const Waypoint = require('waypoints/lib/noframework.waypoints.js')
+
+const WaypointsJS = () => {
```

Comment:
Currently working on a separate branch w/ the beginnings of this. Trying to keep these steps separate as you said, and PR'ing them as needed. If you think there's a better way of going about this process, please let me know!